src/commands/realMouseDown.ts

```diff
@@ -26,6 +27,32 @@ export interface realMouseDownOptions {
   button?: keyof typeof mouseButtonNumbers;
 }
 
+export async function rawMouseDown({
```

Comment:
I've extracted this so the "composite" `realClick` command can benefit from the state tracking